packages/website/components/account/fileUploader/fileUploader.scss

```diff
@@ -0,0 +1,109 @@
+.file-uploader-container {
+  color: $ebony;
+  padding: 3.125rem 8.5rem 3.125rem 8.125rem;
+
+  @include medium {
+    padding: 2.375rem 1.3125rem;
+    width: 100%;
+  }
+
+  > * {
+    position: relative;
+  }
+  h5 {
+    text-align: center;
+    margin-bottom: 0.85rem;
+  }
+}
+
+.file-upload-subheading {
+  @include fontSize_Small;
+  @include fontWeight_Regular;
+  text-align: center;
+  margin-bottom: 1rem;
+
+  a {
+    text-decoration: underline;
+  }
+}
+
+.file-uploader-dropzone {
+  .droparea {
+    padding: 1.875rem 0;
+    background: rgba($white, 0.1);
+    border: solid;
+    border-image: url('../../../assets/images/dashed-border.png') 2 round;
```

Comment:
why not a css border?

Reply:
I think a dashed border was the original intent, but it looked a little asymmetrical as the screen sized down (probably not a huge impact on the design, will try it out on a few different screen sizes).